swig/include/python/python_exceptions.i

```diff
@@ -362,16 +362,19 @@ def UseExceptions():
     """ Enable exceptions in all GDAL related modules (osgeo.gdal, osgeo.ogr, osgeo.osr, osgeo.gnm).
         Note: prior to GDAL 3.7, this only affected the calling module"""
 
+    import importlib.util
```

Comment:
Wouldn't be better to import this at the top level instead of from within the functions?

Reply:
fixed